src/Defra.PTS.Web.UI/Views/Shared/_ErrorSummary.cshtml

```diff
@@ -21,12 +20,11 @@ string title = Thread.CurrentThread.CurrentCulture.EnglishName != "Welsh" ? "The
                     @foreach (var error in errorsFlattened)
                     {
                         <li>
-                            <a href="#@error.field">@error.error</a>
+                            <span class="error-message">@error.error</span>
```

Comment:
This needs to link to the section as per GDS requirements: https://design-system.service.gov.uk/components/error-summary/